src/main/java/mate/academy/dao/UserDao.java

```diff
@@ -0,0 +1,12 @@
+package mate.academy.dao;
+
+import java.util.Optional;
+import mate.academy.model.User;
+
+public interface UserDao {
```

Comment:
```suggestion
@Dao
public interface UserDao {
```
It is necessary for injector

Reply:
Yes? but we need this annotation only in daoImpl

Reply:
I'm so sorry) Yes, it is my mistake)